contract.py

```diff
@@ -541,7 +544,7 @@ class ContractLine(sequence_ordered(), ModelSQL, ModelView):
     service = fields.Many2One('contract.service', 'Service', required=True,
         states={
             'readonly': Bool(Eval('consumptions', [-1])),
-        })
+        }, depends=['consumptions'])
```

Comment:
Why do you add consumptions to depends?

They were removed because otherwise they're loaded in tree view which makes it very slow.

Reply:
consumptions are listed in Pyson statements thus must be in depends.

For the sake to be reassured because I was surprised by this changeset I tested with removed depends: indeed the module didn't work. AFAIR it was not possible to create a new record.